vendor/bat-native-ledger/src/bat/ledger/internal/common/time_util.cc

```diff
@@ -33,4 +35,23 @@ uint64_t GetCurrentTimeStamp() {
   return static_cast<uint64_t>(base::Time::Now().ToDoubleT());
 }
 
+base::TimeDelta GetRandomizedDelay(base::TimeDelta delay) {
+  uint64_t seconds = brave_base::random::Geometric(delay.InSecondsF());
+  return base::TimeDelta::FromSeconds(static_cast<int64_t>(seconds));
+}
+
+base::TimeDelta GetRandomizedDelayWithBackoff(
+    base::TimeDelta delay,
+    base::TimeDelta max_delay,
+    int backoff_count) {
+  for (int i = 0; i < backoff_count; ++i) {
+    delay *= 2;
```

Comment:
nit: use shift operator `delay << backoff_count`

Reply:
`delay` here is a `base::TimeDelta`, which doesn't support overloading for shift operators (other than `ostream` of course).

Also, with shifting we'd need to be careful to not shift too far (we'd have to range check `backoff_count`).

Reply:
but we could multiple delay by `1 << backoff_count` to remove the loop. and cap backoff_count to a maximum delay

Reply:
nice @tmancey - will do.